src/components/AccessibleSection.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import image1 from '../assets/image1.jpg'
+
+const AccessibleSection: React.FC = () => {
+ return (
+    <>
```

Comment:
rimuoviamo il fragment essendo inutile in questo casso, siccome ha un solo child